tests/test_room.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from classes.room import Room
+from classes.dojo import Dojo
+
+
+class RoomClassTest(unittest.TestCase):
+    def test_add_occupant(self):
+        my_class_instance = Room("Blue", "office", 6)
+        my_dojo_instance = Dojo()
+        new_fellow = my_dojo_instance.add_person("fellow", "Peter", "Musonye", "Y")
+        initial_persons_count = len(my_class_instance.persons)
+        new_occupant = my_class_instance.add_occupant(new_fellow)
+        # self.assertTrue(new_occupant)
+        new_persons_count = len(my_class_instance.persons)
+        self.assertEqual(new_persons_count - initial_persons_count, 1)
+
+    def test_cannot_add_more_than_max_occupants(self):
+        my_class_instance = Room("Blue", "office", 4)
+        my_dojo_instance = Dojo()
+        fellow_1 = my_dojo_instance.add_person("fellow", "Peter", "Musonye", "Y")
+        fellow_2 = my_dojo_instance.add_person("staff", "Farhan", "Abdi")
```

Comment:
Having a person of type `staff` stored in a variable called `fellow_2` is confusing. Perhaps you meant for them to be a fellow.